libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -714,21 +714,36 @@ static void logConnackResponse( uint8_t responseCode )
 
     assert( responseCode <= 5 );
 
-    /* Declare the CONNACK response code strings. The fourth byte of CONNACK
-     * indexes into this array for the corresponding response. This array
-     * does not need to be allocated if logs are not enabled. */
-    #if LIBRARY_LOG_LEVEL > LOG_NONE
-        static const char * const pConnackResponses[ 6 ] =
-        {
-            "Connection accepted.",                               /* 0 */
-            "Connection refused: unacceptable protocol version.", /* 1 */
-            "Connection refused: identifier rejected.",           /* 2 */
-            "Connection refused: server unavailable",             /* 3 */
-            "Connection refused: bad user name or password.",     /* 4 */
-            "Connection refused: not authorized."                 /* 5 */
-        };
-        LogErrorWithArgs( "%s", pConnackResponses[ responseCode ] );
-    #endif /* if LIBRARY_LOG_LEVEL > LOG_NONE */
+    /* Log an error based on the CONNACK response code. */
+    switch( responseCode )
+    {
+        case 0u:
+            LogError( "Connection accepted." );
```

Comment:
If each response code gets its own log statement then let's take the opportunity to change this to `LogInfo`, since it's indicating success.

Reply:
Okay, strange that it was logged as error earlier.

Reply:
Changed to `LogInfo`